src/lib/create-from-script-tag.js

```diff
@@ -1,11 +1,31 @@
 'use strict';
 
+var assign = require('./assign').assign;
 var find = require('./find-parent-form');
 var uuid = require('./uuid');
 var DropinError = require('./dropin-error');
 
+function addCompositeKeyValuePairToObject(obj, key, value) {
+  var decomposedKeys = key.split('.');
```

Comment:
Can we do `var rootKey = decomposedKeys[0];`?

Looks like we use the first first element throughout this function.